torchrl/envs/libs/vmas.py

```diff
@@ -17,7 +17,7 @@
 )
 from torchrl.envs.common import _EnvWrapper, EnvBase
 from torchrl.envs.libs.gym import _gym_to_torchrl_spec_transform, set_gym_backend
-from torchrl.envs.utils import _selective_unsqueeze
+from torchrl.envs.utils import _classproperty, _selective_unsqueeze
 
 IMPORT_ERR = None
 try:
```

Comment:
Global import.

Probably we can use the better module checking here and remove the global import.

Happy to do this, i think i can push here,

Let me know

Reply:
Oh no that's ok i forgot that one